apps/backend/src/libs/modules/mail/mail.module.ts

```diff
@@ -62,10 +45,25 @@ class Mail {
 		subject: string;
 		text: string;
 	}): Promise<boolean> {
-		const from = `${this.senderName} <${this.senderEmail}>`;
+		const sendEmailCommand = new SendEmailCommand({
+			Destination: {
+				ToAddresses: [to],
```

Comment:
```suggestion
				ToAddresses: [email],
```
and remove renaming `email: to`